.github/workflows/contracts.yaml

```diff
@@ -27,8 +39,135 @@ jobs:
+            @keep-network/keep-core@1.8.0-ropsten.13+main.6a98f2b066bc27910ab1715e7d20b7ba3c3ff21e
+
+      - name: Configure tenderly
+        # if: github.event.inputs.environment == 'ropsten' TODO: uncomment
+        env:
+          TENDERLY_TOKEN: ${{ secrets.TENDERLY_TOKEN }}
+        run: ./config_tenderly.sh
+
+      - name: Deploy contracts
+        env:
+          CHAIN_API_URL: ${{ secrets.ROPSTEN_ETH_HOSTNAME_HTTP }}
+          CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY: ${{ secrets.ROPSTEN_ETH_CONTRACT_OWNER_PRIVATE_KEY }}
+        # run: yarn deploy --network ${{ github.event.inputs.environment }} TODO: uncomment, remove below file
+        run: yarn deploy --network ropsten
+
+      - name: Bump up package version
+        continue-on-error: true # remove before merging to main
+        id: npm-version-bump
+        uses: keep-network/npm-version-bump@v2
+        with:
+          # environment: ${{ github.event.inputs.environment }}
+          environment: ropsten
+          branch: ${{ github.ref }}
+          commit: ${{ github.sha }}
+
+      - name: Publish to npm # TODO: switch off --dry-run
+        continue-on-error: true # remove before merging to main
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        # run: npm publish --access=public --network=${{ github.event.inputs.environment }} --dry-run TODO: uncomment, remove below
+        run: npm publish --access=public --network=ropsten --dry-run
+
+      - name: Upload files needed for etherscan verification
+        uses: actions/upload-artifact@v2
+        with:
+          name: Artifacts for etherscan verifcation
+          path: |
+            ./deployments
+            ./package.json
+            ./yarn.lock
+
+  contracts-etherscan-verification:
+    needs: [contracts-deployment-testnet]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Download files needed for etherscan verification
+        uses: actions/download-artifact@v2
+        with:
+          name: Artifacts for etherscan verifcation
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+          cache: "yarn"
+
+      - name: Install needed dependencies
+        run: yarn install --frozen-lockfile
+
+      # If we don't remove the `keep-core` contracts from `node-modules`, the
+      # `etherscan-verify` plugins tries to verify them, which is not desired.
+      - name: Prepare for verification on Etherscan
+        run: |
+          rm -rf ./node_modules/@keep-network/keep-core
+
+      - name: Verify contracts on Etherscan
+        env:
+          ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_API_KEY }}
+          CHAIN_API_URL: ${{ secrets.ROPSTEN_ETH_HOSTNAME_HTTP }}
+        # run: |
+        #   yarn run hardhat --network ${{ github.event.inputs.environment }} \
+        #     etherscan-verify --license MIT
+        run: |
+          yarn run hardhat --network ropsten \
+            etherscan-verify --license MIT
```

Comment:
@Kuba, `T` and `VendingMachine` contracts use `GPL-3.0-or-later` licens, which is not supported by etherscan. Do you think I could force `MIT license here (`yarn run hardhat --network ropsten etherscan-verify --license MIT --force-license`)? I verified that it works locally. Without the `--force-license` or `--license MIT --force-license` I was getting errors preventing from verification).

Reply:
Looks like GPL v3 should be supported by etherscan: https://etherscan.io/contract-license-types.
I'm not sure what value the command is expecting though.

Reply:
I have force using `GPL-3.0` license (`yarn run hardhat --network ropsten etherscan-verify --license GPL-3.0 --force-license`), see cfaa9e1.